style.css

```diff
@@ -0,0 +1,111 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500&display=swap");
+
+
+.mainBlock {
+    display: flex;
+    justify-content: center;
+    position: absolute;
+    height: 600px;
+    width: 600px;
+    background-color: rgb(219, 95, 23);
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 15px;
+    color: white;
+    font-family: 'Montserrat';
+    font-style: normal;
+
+}
+
+/* profile */
+
+.profileMenu {
+    float: left;
+    width: 50%; 
+    background: lightsalmon;
+    border-radius: 15px;
+    margin: 15px;
+    padding: 15px;
+    text-align: center;
+}
+img {
+    border-radius: 100px;
+    transition: all 0.3s ease-in-out 0s;
+}
+img:hover {
+    transform: rotate(360deg);
+    transition: all 0.3s ease-in-out 0s;
+}
+.profileName {
```

Comment:
add a bold style to this text, and try to add Uppercase letters
you cant use `text-transform`